src/LogEntry.php

```diff
@@ -57,8 +57,8 @@ public function isOfLogLevel(array $logLevelList)
 
     public function containsRegExp($regExp)
     {
-        Assertion::string($regExp);
-        Assertion::minLength($regExp, 1);
+        Assert::string($regExp);
+        Assert::minLength($regExp, 1);
```

Comment:
Could be of minLength of 2 .. can’t think about a valid regex of length 1 that makes sense in that situation.

Reply:
👍